trade-finance-manager-ui/templates/index.njk

```diff
@@ -2,7 +2,7 @@
 <html lang="en" class="govuk-template ">
   <head>
     <meta charset="utf-8" />
-    <title{% if pageTitleLang %} lang="{{ pageTitleLang }}"{% endif %}>{% block pageTitle %}{% endblock %} - TFM</title>
+    <title{% if pageTitleLang %} lang="{{ pageTitleLang }}"{% endif %}>{% block pageTitle %}{% endblock %}, Trade Finance Manager, gov.uk</title>
```

Comment:
Multiline formatting please.

Reply:
Not sure if we need multiline in <html><head><title>, but looks like it works fine.